cmd/starter/c/starter.c

```diff
@@ -1413,7 +1413,7 @@ __attribute__((constructor)) static void init(void) {
         if ( sconfig->starter.isSuid ) {
             priv_escalate(true);
         } else if ( uid != 0 ) {
-            fatalf("No setuid installation found, for unprivileged installation use: ./mconfig --without-suid\n");
+            fatalf("No setuid installation found\n");
```

Comment:
Can you revert to `No setuid installation found, for privileged installation use: ./mconfig --with-suid\n` ?